packages/monitoring-system/data-processor/test/unit/data-processors/github-data-processor.spec.ts

```diff
@@ -38,12 +38,17 @@ let mockFirestoreData1: FirestoreData;
 
 function resetMockData() {
```

Comment:
I changed `loadFixture`'s `copy` param to be `false` by default instead of true so had to adjust parameters here.